pkg/utils/constant.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+const (
+	PortNo  = ":8080"
+	HMDB    = "HMDB"
+	HMDB_TEST ="HMDB_TEST"
+	Doctors = "Doctors"
```

Comment:
It can be renamed to something like `DoctorCollection = "docters"`, the collection name should not start with a capital letter.